stl/inc/system_error

```diff
@@ -6,14 +6,16 @@
 #pragma once
 #ifndef _SYSTEM_ERROR_
 #define _SYSTEM_ERROR_
-#include <yvals_core.h>
+#include <yvals.h>

```

Comment:
@CaseyCarter do you know why this very-non-Core header said `<yvals_core.h>`?

Reply:
A core header cannot include `yvals.h`, but a non-core header may (and in fact certainly does indirectly) include `yvals_core.h`. When I added the `_STL_COMPILER_PREPROCESSOR` guards, I spammed
```c++
#include <yvals_core.h>
#if _STL_COMPILER_PREPROCESSOR
// ...
#endif // _STL_COMPILER_PREPROCESSOR
```
into all of the STL headers, and then cut-and-pasted any other `#include<yvals` line atop the newly-added `#include <yvals_core.h>`. We weren't concerned at the time with creating a "core/non-core headers always directly include yvals/yvals_core" invariant.

Reply:
I see, I'm going to leave this at yvals then.